Integrations/Tenable_sc/Tenable_sc.yml

```diff
@@ -1,62 +1,61 @@
+category: Vulnerability Management
 commonfields:
   id: Tenable.sc
   version: -1
-name: Tenable.sc
-display: Tenable.sc
-category: Vulnerability Management
-description: With Tenable.sc (formerly SecurityCenter) you get a real-time, continuous
-  assessment of your security posture so you can find and fix vulnerabilities faster.
 configuration:
 - display: Server URL (e.g. https://192.168.0.1)
   name: server
-  defaultvalue: ""
-  type: 0
   required: true
+  type: 0
 - display: Username
   name: credentials
-  defaultvalue: ""
-  type: 9
   required: true
+  type: 9
 - display: Trust any certificate (unsecure)
   name: unsecure
-  defaultvalue: ""
-  type: 8
   required: false
+  type: 8
 - display: Use system proxy settings
   name: proxy
-  defaultvalue: ""
-  type: 8
   required: false
+  type: 8
 - display: Fetch incidents
   name: isFetch
-  defaultvalue: ""
-  type: 8
   required: false
+  type: 8
 - display: Incident type
   name: incidentType
-  defaultvalue: ""
-  type: 13
   required: false
-- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
+  type: 13
+- defaultvalue: 3 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
```

Comment:
This design seems tricky to me.
However, let's review it after we understand if this is even needed

Reply:
@dantavori it still seems tricky to me, I think we should review it and consider removing it since it isn't used